scene/3d/node_3d.h

```diff
@@ -68,20 +71,36 @@ class Node3D : public Node {
 	};
 
 private:
+	// For the sake of ease of use, Node3D can operate with Transforms (Basis+Origin), Quaterinon/Scale and Euler Rotation/Scale
+	// Transform and Quaterinon are stored in data.local_transform Basis (so quaternion is not really stored, but converted back/fortm from 3x3 matrix on demand).
+	// Euler needs to be kept separate because converting to Basis and back may result in a different vector (which is troublesome for users
+	// editing in the inspector, not only because of the numerical precision loss but because they expects these rotations to be consistent, or support
```

Comment:
```suggestion
	// editing in the inspector, not only because of the numerical precision loss but because they expect these rotations to be consistent, or support
```